models/consultation.py

```diff
@@ -1,9 +1,9 @@
 class ConsultationTemplate:
     @staticmethod
     def convert(consultation):
-        return f"Учитель {consultation.teacher} \nКабинет: {consultation.room}\nПочта: {consultation.email}" \
+        return f"👨‍🏫 Учитель {consultation.teacher} \nКабинет: {consultation.room}\n✉ Почта: {consultation.email}" \
```

Comment:
Teacher's email not everywhere displayed and written. I recommend to add some if statement for this.